lib/index.js

```diff
@@ -1,13 +1,16 @@
-const Source = require('./source');
```

Comment:
Not sure if having `;` is bad

Reply:
http://blog.namangoel.com/considering-the-npm-styleguide

Reply:
http://blog.namangoel.com/considering-the-npm-styleguide

Reply:
http://blog.namangoel.com/considering-the-npm-styleguide